gammapy/utils/random/tests/test_inverse_cdf.py

```diff
@@ -0,0 +1,65 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+import scipy.stats as stats
+from numpy.testing import assert_allclose
+
+from ..inverse_cdf import InverseCDFSampler
+
+
+def uniform_dist(x, a, b):
+    return np.select([x <= a, x >= b], [0, 0], 1 / (b - a))
+
+
+def gauss_dist(x, mu, sigma):
+    return stats.norm.pdf(x, mu, sigma)
+
+
+def test_uniform_dist_sampling():
+    n_sampled = 1000
+    x = np.linspace(-2, 2, n_sampled)
+
+    a, b = -1, 1
+    pdf = uniform_dist(x, a=a, b=b)
+    sampler = InverseCDFSampler(pdf=pdf, random_state=0)
+
+    idx = sampler.sample(int(1e4))
+    x_sampled = np.interp(idx, np.arange(n_sampled), x)
+
+    assert_allclose(np.mean(x_sampled), 0.5 * (a + b), atol=0.01)
+    assert_allclose(
+        np.std(x_sampled), np.sqrt(1 / 3 * (a ** 2 + a * b + b ** 2)), rtol=0.01
+    )
+
+
+def test_norm_dist_sampling():
+    n_sampled = 1000
+    x = np.linspace(-2, 2, n_sampled)
+
+    mu, sigma = 0, 0.1
+    pdf = gauss_dist(x=x, mu=mu, sigma=sigma)
+    sampler = InverseCDFSampler(pdf=pdf, random_state=0)
+
+    idx = sampler.sample(int(1e5))
+    x_sampled = np.interp(idx, np.arange(n_sampled), x)
+
+    assert_allclose(np.mean(x_sampled), mu, atol=0.01)
+    assert_allclose(np.std(x_sampled), sigma, atol=0.005)
+
+
+def test_norm_dist_sampling():
```

Comment:
You put two test functions with the same name: `test_norm_dist_sampling`

What will happen is that the second `def` statement replaces the first one, and the first test will never run.
Please rename the test functions to be unique.

Reply:
Sorry, this is a mistake. It will be changed soon.